Library/Formula/ruby-build.rb

```diff
@@ -3,8 +3,8 @@
 class RubyBuild < Formula
   head "https://github.com/sstephenson/ruby-build.git"
   homepage "https://github.com/sstephenson/ruby-build"
-  url "https://github.com/sstephenson/ruby-build/archive/v20150303.tar.gz"
-  sha1 "c15c1f6613d6c518544ed0d90a0cf33eefc7d73f"
+  url "https://github.com/sstephenson/ruby-build/archive/v20150319zf.tar.gz"
+  sha1 "31c2e082b3fc82e90f86ff787184ecf0f9ace659"
 
```

Comment:
brew is having trouble parsing the version from the URL; can you add `version "20150319zf"`, please?